angular-ngrx-scss/src/app/state/profile/profile.selectors.ts

```diff
@@ -0,0 +1,11 @@
+import { createFeatureSelector, createSelector } from '@ngrx/store';
+import { ProfileState } from './profile.state';
+
+export const profileFeatureKey = 'profile';
+export const selectProfileState =
+  createFeatureSelector<ProfileState>(profileFeatureKey);
+
+export const selectProfile = createSelector(
+  selectProfileState,
+  (state: ProfileState) => state,
+);
```

Comment:
Mentioned in another comment, but I'd add a selector here for the repos.

Reply:
Would these selectors be used by this PR at all?